### DISCUSSION TITLE
Fail assets open

### DISCUSSION THREAD

**Discussion by DidoNael** [requarks/wiki#6507] [Help / Questions]
Hello friends, even giving permission to view my groups, they cannot view an image that is highlighted as home, the image simply breaks. Can you help?

![image](https://github.com/requarks/wiki/assets/52351947/44e1bb37-b2e3-4a53-8fc0-57e039b442ae)

Informations: 
![image](https://github.com/requarks/wiki/assets/52351947/f79fd564-4d88-4210-a5fd-7dc0f140c8b8)

![image](https://github.com/requarks/wiki/assets/52351947/01b0be27-febf-4d3a-8067-a893a1f7d5c3)

![image](https://github.com/requarks/wiki/assets/52351947/4066b99a-1a49-4250-8ffd-d41b31efd9ed)

**Comment by P4mes4n** [comment]
Your User Group only has access to the `/home` path. This also applies to the asset path. My guess is you placed the asset in the root directory (as is the default) and as such this group can not see it.
Create a Folder for Assets named "home" at root level and put the image there. Then it should work.

**Reply by DidoNael** [reply]
Obrigado, a sua orientação funcionou perfeitamente.
